app/routes.js

```diff
@@ -102,7 +102,7 @@ module.exports.paths = paths
 
 module.exports.bind = function (app) {
   const account = new Router({ mergeParams: true })
-  account.use(getServiceAndAccount, userIsAuthorised)
+  account.use(getServiceAndAccount, userIsAuthorised, validateAndRefreshCsrf)
```

Comment:
Looks like we need `ensureSessionHasCsrfSecret` where creates new csrfSecret if one doesn't exist on session.

```suggestion
  account.use(getServiceAndAccount, userIsAuthorised, ensureSessionHasCsrfSecret, validateAndRefreshCsrf)
```

Reply:
Just updated it - thanks.